packages/core/src/__tests__/ceramic-api.test.ts

```diff
@@ -435,7 +433,7 @@ describe('Ceramic API', () => {
       expect(Object.keys(streams).length).toEqual(6)
     })
 
-    it('can load streams for array of multiqueries even if streamid or path throws error', async () => {
+    it.skip('can load streams for array of multiqueries even if streamid or path throws error', async () => {
```

Comment:
is this skipped purposefully?

Reply:
Ooh, good catch. Should not have skipped it.